frontend/src/components/CriteriaBarChart.tsx

```diff
@@ -153,7 +163,7 @@ const CriteriaBarChart = ({ video }: Props) => {
   // ResponsiveContainer adds the width and height props to its child component.
   // We need the width to position the icons.
   return (
-    <ResponsiveContainer width="100%" aspect={1}>
+    <ResponsiveContainer width="100%" height={500}>
```

Comment:
@lfaucon I switched back to a fixed height because otherwise on some screen size the icon is cropped.

Reply:
I changed that in d03d117767202a8c519ae67291ecae602cebe0c0 because the height should be calculated from the criteria count. This change could also be a separate PR though.

Reply:
These changes have been made in `main` (a50885df52296d75ae1d7a73994891a4c819d860).